test/unit/bls_test.exs

```diff
@@ -0,0 +1,32 @@
+def BlsTest do
+
+  use ExUnit.Case
+  use Rustler, otp_app: :lambda_ethereum_consensus, crate: "bls_nif"
+  alias Bls.key_validate, as: key_validate
```

Comment:
We don't import functions like this, since the module name adds a lot of context.

```suggestion
```